providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/common/ChannelConnector.java

```diff
@@ -89,19 +74,6 @@ public void initialize() throws Exception {
         monitorChannelState(ConnectivityState.READY);
     }
 
-    /**
-     * Returns the blocking service stub for making blocking GRPC calls.
-     *
-     * @return the blocking service stub
-     */
-    public K getBlockingStub() {
-        K stub = blockingStubFunction.apply(channel).withWaitForReady();
-        if (this.deadline > 0) {
-            stub = stub.withDeadlineAfter(this.deadline, TimeUnit.MILLISECONDS);
-        }
-        return stub;
-    }
-
```

Comment:
Refactor: as pointed out by @aepfli , this has no business being in this class,, which other than this, only handles channel state. I've moved this stubs into their respective correct places which already have the async stubs.